tests/import_test.go

```diff
@@ -189,6 +189,63 @@ var _ = Describe("[rfe_id:1115][crit:high][vendor:cnv-qe@redhat.com][level:compo
 			Expect(importer.DeletionTimestamp).To(BeNil())
 		}
 	})
+
+	It("[test_id:6689]succeed importing VDDK data volume with extra arguments ConfigMap set", Label("VDDK"), func() {
```

Comment:
Would you mind putting this under a separate describe so it doesn't inherit the "Serial" part?
(I think this test can run in parallel to others just fine)

You can also change `[test_id:6689]` to `[test_id:XXXX]`; I don't think it works well if this entry is missing from the ID database